scripts-available/CDB_Overviews.sql

```diff
@@ -1,3 +1,29 @@
+-- Maximum zoom level for which overviews may be created
+CREATE OR REPLACE FUNCTION _CDB_MaxOverviewLevel()
```

Comment:
why not changing the internal representation of coordinates to `bigint` to be able to hold them up to z = 31?

functions such as `CDB_XYZ_Extent` and others can have their `bigint` counterpart. Both can coexist and the `integer` version be marked as "deprecated" and make them issue a warning.